vm/src/stdlib/time.rs

```diff
@@ -91,15 +91,19 @@ mod time {
 
     #[cfg(all(target_arch = "wasm32", not(target_os = "wasi")))]
     fn _time(_vm: &VirtualMachine) -> PyResult<f64> {
-        use wasm_bindgen::prelude::*;
-        #[wasm_bindgen]
-        extern "C" {
-            type Date;
-            #[wasm_bindgen(static_method_of = Date)]
-            fn now() -> f64;
+        #[cfg(feature = "wasmbind")]
+        {
+            use wasm_bindgen::prelude::*;
+            #[wasm_bindgen]
+            extern "C" {
+                type Date;
+                #[wasm_bindgen(static_method_of = Date)]
+                fn now() -> f64;
+            }
+            // Date.now returns unix time in milliseconds, we want it in seconds
+            return Ok(Date::now() / 1000.0);
         }
-        // Date.now returns unix time in milliseconds, we want it in seconds
-        Ok(Date::now() / 1000.0)
+        panic!("Date not available")
```

Comment:
Why panic?

Reply:
If you are compiling wasm32-unknown-unknown and not wasi then std time doesn't work, so there is no way to get a date generally as far as I know for for wasm32-unknown-unknown.

Reply:
In our project, which must compile and run in a wasm32-unknown-unknown environment, std time doesn't work, and we have our own time implementation that we import from the host environment. We would love for a solution to this problem as well, but this PR is scoped for general use.

Reply:
When the interpreter panics, the interpreter process comes to a complete emergency stop. From there, you have to start it again. Is this what you want?

Reply:
Typically, unsupported platforms in Python just don't expose the module/function/constant etc (the `os` module is an example of this). If the `time` module doesn't work for the target it would make more sense to just not have it be available.

Baring that, an exception would be the second best option.

Reply:
I also vote for hiding the function `time.time` when not available.
Then user can set it like `time.time = custom_time`

Reply:
Ah, I found `process_time` and `process_time_ns` raises error. `NowImplementedError` also will be ok.